templates/azure/index.html

```diff
@@ -25,61 +31,66 @@ <h1>Ubuntu on Azure</h1>
           <p>
             Beyond Ubuntu Server images, Canonical offers a number of premium Ubuntu images that extend Ubuntu Server with additional security and management features, billed through your existing Microsoft Azure account.
           </p>
-          <a href="/azure/contact-us" class="p-button--positive js-invoke-modal">Talk to us about Ubuntu on Azure</a>
         </div>
-        <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
+        <hr class="is-muted" />
+        <a href="/azure/contact-us" class="p-button--positive js-invoke-modal">Talk to us about Ubuntu on Azure</a>
+      </div>
+      <div class="col">
+        <div class="p-image-container--16-9 is-highlighted">
           {{ image(url="https://assets.ubuntu.com/v1/da9a1344-Microsoft-Azure-logo_stacked_transparent.png",
                     alt="",
-                    width="350",
-                    height="100",
+                    width="168",
+                    height="49",
                     hi_def=True,
-                    loading="auto",) | safe
+                    loading="auto",
+                    attrs={"class": "p-image-container__image"}) | safe
           }}
         </div>
       </div>
     </div>
   </section>
 
-  <section class="p-strip--light">
-    <div class="row">
-      <div class="col-12">
-        <h2 class="p-muted-heading">Ubuntu on Azure powers cloud leaders like</h2>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule--muted" />
+      <div class="col p-section--shallow">
+        <h2>
+          Ubuntu on Azure powers
+          <br />
+          cloud leaders
```

Comment:
in the copy doc it says "...leaders like"

Reply:
Double checked with Juan and the doc was just out of date, the "like" was supposed to be dropped. He's updated the doc